main/sw/inc/shellio.hxx

```diff
@@ -414,18 +410,18 @@ SW_DLLPUBLIC SwRead SwGetReaderXML();
 // END source/filter/basflt/fltini.cxx
 
 
-extern sal_Bool SetHTMLTemplate( SwDoc &rDoc ); //Fuer Vorlagen aus HTML.vor laden shellio.cxx
+extern sal_Bool SetHTMLTemplate( SwDoc &rDoc ); // HTML templates (pre?)load shellio.cxx
```

Comment:
Or "before loading"? I have no idea :-(

Reply:
I am not sure, what the german comment means. It could mean before loading or only loading. :(

Reply:
I can find other occurrences of the string `HTML.vor` in comments, always besides a call to `SetHTMLTemplate()`. Maybe it is an abbreviation for "HTML template"? A former file extension? Just speculating...

Reply:
I slowly get the Idea that the comment is borked. I checked opengrok and I found the line:
 bHTMLTemplSet = SetHTMLTemplate( *GetDoc() );//Styles aus HTML.vor in InitNew()  (see docshini.cxx 127)


I assume from the code that it should mean:
Extract HTMLTemplates from given Document.

But when I check docsh2.cxx 1720 :
// fix #51032#: Die HTML-Vorlage muss noch gesetzt werden
SetHTMLTemplate( *GetDoc() );	//Styles aus HTML.vor

The fix comment translates to: The HTML-Template has to be set
The next line says Syles from HTML.vor maybe that is a fixed term?

Reply:
Great. Checking the Issue is leading to a wrong issue. Probably the Number is to old to be recognized?

Reply:
I'd just translate "HTML.vor" into "HTML template" and carry on

Reply:
"Load template from HTML template shellio.cxx" ? Does not make sense to me. :(

Reply:
I think that "laden shellio.txt" means "the code is loaded from shellio.cxx". "Laden" refers to the code, not to the template data. In fact, the function is defined in file main/sw/source/filter/basflt/shellio.cxx
Just my opinion, of course

Reply:
should be fixed now.